packages/chrome/demo/stories/SheetStory.tsx

```diff
@@ -23,14 +23,18 @@ interface ISheetComponentProps extends ISheetProps {
   footerItems: IFooterItem[];
   hasHeader: boolean;
   isCompact: boolean;
+  hasTitle?: boolean;
   title: string;
+  hasDescription?: boolean;
```

Comment:
I think you can guarantee that Storybook is going to give you a `boolean` and not `undefined`.

Reply:
The reason it was optional is that this story is re-used by the `Chrome` story and those two controls are not present there.

Reply:
gotcha. Thanks for the clarification.